tests/test_date.nit

```diff
@@ -0,0 +1,13 @@
+import date
```

Comment:
This can be moved in the Date class comment as a `docunit`. See http://nitlanguage.org/tools/nitunit/.

Example:
~~~
# The main class Date (or a better synopsis)
#
#     var d  = new Date(2014, 01, 09)
#     assert d.to_s == "2014/01/09"
class Date
...
~~~